examples/assemblyscript/package.json

```diff
@@ -5,8 +5,8 @@
   "main": "src/main.ts",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "asc --target release -Osize",
-    "build:debug": "asc --target debug"
+    "build": "asc --exportRuntime --target release -Osize",
+    "build:debug": "asc --exportRuntime --target debug"
```

Comment:
Ah, that makes sense. Thanks so much, @konsumer! This is awesome.

Reply:
You might be able to get away with no runtime somehow, but I'm not totally sure. It seems like just accepting that assemblyscript will add a few things to your wasm, and adding `abort` to the host runtime seems to be an ok middleground that doesn't require too much language-lockin with assmeblyscript. It seems useful to have a "fatal error" kind of function anyway, otherwise the wasm will just seize without any info.